tsl/src/bgw_policy/drop_chunks_api.c

```diff
@@ -46,15 +46,15 @@
 										  Float8GetDatum(0)))
 /* Right now, there is an infinite number of retries for drop_chunks jobs */
 #define DEFAULT_MAX_RETRIES -1
-/* Default retry period for drop_chunks_jobs is currently 12 hours */
+/* Default retry period for drop_chunks_jobs is currently 5 minutes */
 #define DEFAULT_RETRY_PERIOD                                                                       \
 	DatumGetIntervalP(DirectFunctionCall7(make_interval,                                           \
```

Comment:
i think you should use interval_in instead of make_interval

Reply:
Done, and fixed elsewhere as well.

Reply:
much better